### DISCUSSION TITLE
Sections

### DISCUSSION THREAD

**Discussion by alfreditog** [hirasso/scrollmirror#21] [Show and tell]
Hello,
Is it possible to configure the sync scroll based on sections or chapters? I mean for example  window 1 with 3 sections, window 2 the same, and the scroll be based on the beginning and end of section 1 then pass to section 2 and so on for both windows? Thanks in advance.

**Comment by hirasso** [comment]
Something like this is not possible and not planned with ScrollMirror. I'm trying to keep it simple both with the functionality as well as with the API.

**Comment by pablospe** [comment]
> Hello,
> Is it possible to configure the sync scroll based on sections or chapters? I mean for example  window 1 with 3 sections, window 2 the same, and the scroll be based on the beginning and end of section 1 then pass to section 2 and so on for both windows? Thanks in advance.

Have you found a solution for this?

**Comment by hirasso** [comment]
As this is not in the scope of scrollmirror I'll go ahead and close this discussion.
